include/fmt/base.h

```diff
@@ -1484,6 +1492,20 @@ template <typename Context> struct arg_mapper {
   FMT_MAP_API auto map(double val) -> double { return val; }
   FMT_MAP_API auto map(long double val) -> long double { return val; }
 
+#if FMT_USE_BITINT
+#  pragma clang diagnostic push
+#  pragma clang diagnostic ignored "-Wbit-int-extension"
+  template <class T, int N = 0> struct is_bitint : std::false_type {};
+  template <int N> struct is_bitint<_BitInt(N)> : std::true_type {};
+  template <int N> struct is_bitint<unsigned _BitInt(N)> : std::true_type {};
+
+  template <class T, FMT_ENABLE_IF(is_bitint<remove_cvref_t<T>>::value)>
+  FMT_MAP_API auto map(T&& val) -> decltype(val) {
```

Comment:
`_BitInt` should be converted to appropriate integer type.

Reply:
Are we assuming the maximum width for a `_BitInt` is 128 bits here, so we can widen to any of the native integer types (including `__int128`)?

I think [officially](https://releases.llvm.org/14.0.0/tools/clang/docs/ReleaseNotes.html#c-language-changes-in-clang) this is what clang supports according to `BITINT_MAXWIDTH` for c++

Although for C(23), it supports up to 8388608
https://godbolt.org/z/jzfGYoY6a

Reply:
I think we can assume that for now and in any case for smaller sizes we should convert to integers for efficiency instead of going through the formatter extension.